src/config/constants/nfts/types.ts

```diff
@@ -0,0 +1,39 @@
+import { Address, Images } from '../types'
+
+export enum CollectionKey {
+  PANCAKE = 'pancake',
+}
+
+type Collection = {
+  name: string
+  address: Address
+}
+
+export type Collections = {
+  [key in CollectionKey]: Collection
+}
+
+export type NftImages = {
+  blur?: string
+} & Images
+
+export type Nft = {
+  id?: number | string
+  name: string
+  description: string
+  images: NftImages
+  video?: {
+    webm: string
+    mp4: string
+  }
+
+  // Uniquely identifies the nft.
+  // Used for matching an NFT from the config with the data from the NFT's tokenURI
+  identifier: string
```

Comment:
We  can probably remove this and use the `id` now.

Reply:
Not a priority of course.

Reply:
It will change the state (a big rework), and this code will be removed on the nft market anyway